app/src/main/java/org/oppia/android/app/player/state/StatePlayerRecyclerViewAssembler.kt

```diff
@@ -1117,7 +1117,7 @@ class StatePlayerRecyclerViewAssembler private constructor(
       supportsConceptCards: Boolean
     ): BindableAdapter<StringList> {
       return BindableAdapter.SingleTypeBuilder
-        .newBuilder<StringList>()
+        .Factory(fragment).create<StringList>()
```

Comment:
For this & the one below: we should be using an injected factory rather than constructing a new one.

Reply:
Patched.